lib/matplotlib/tests/test_axes.py

```diff
@@ -351,7 +349,7 @@ def test_shaped_data():
     plt.plot(y2)
 
     plt.subplot(413)
-    assert_raises(ValueError, plt.plot, (y1, y2))
+    pytest.raises(ValueError, plt.plot(y1, y2))
```

Comment:
I prefer the `raises` context manager:

https://pytest.org/latest/assert.html#assertions-about-expected-exceptions

```
def test_zero_division():
    with pytest.raises(ZeroDivisionError):
        1 / 0
```

I think it keeps the code being run looking like code.